internal/runtime/test/v1alpha1/fake_types.go

```diff
@@ -72,6 +72,10 @@ type FakeResponse struct {
 	First  int
 }
 
+func (in *FakeResponse) DeepCopyObject() runtime.Object {
```

Comment:
Q: What do you think about generating the deepcopy stuff for the fake types the same way that we generate conversion? It's probably simpler to adjust the Makefile a bit vs. adding the funcs over time with panics :) 

If I see correctly:
```
		paths=./internal/runtime/test/v1alpha1 \
		paths=./internal/runtime/test/v1alpha2 \
```
to generate-go-deepcopy-core